Update to Java 17

LadyCailin opened the pull request:
Updates to Java 17 and Bungee 1.20.

LadyCailin committed:
Update to Java 17

roblabla commented:
Hey, thanks for this. Unfortunately, I haven't maintained this for a very long time, so I'm in no position to test or publish this.

Is this update necessary for MoveMeNow to work?

LadyCailin commented:
It is sufficient (if not necessary) to fix it. There were several plugins that had the same error, of which updating to Java 17 fixed it. I can't recall the exact error now, but it was some kind of Error which would normally have been a compile error. In any case, all of the plugins I fixed that had this error I fixed the same way. Perhaps there was a different/better way to fix them, but bringing it up to Java 17 is anyways minimal impact.
